OracleWebLogic/samples/12213-webtier-apache/container-scripts/run-httpd.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+# Apache gets grumpy about PID files pre-existing
+rm -f /run/httpd/httpd.pid
+
+exec httpd -DFOREGROUND
+
+tail -f /var/log/httpd/access_log
```

Comment:
Why are you doing this if you're running `httpd` in `FOREGROUND` mode? This shouldn't be necessary (and is not used in the upstream `httpd` image either).

Reply:
fixed